src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -4378,20 +4334,15 @@ private void addJavadocOptions(
 
         // all options in alphabetical order
 
-        if (old && isJavaDocVersionAtLeast(SINCE_JAVADOC_1_4)) {
-            if (getLog().isWarnEnabled()) {
-                getLog().warn("Javadoc 1.4+ doesn't support the -1.1 switch anymore. Ignore this option.");
-            }
-        } else {
-            addArgIf(arguments, old, "-1.1");
+        if (getLog().isWarnEnabled()) {
+            getLog().warn("Javadoc 1.4+ doesn't support the -1.1 switch anymore. Ignore this option.");
         }
```

Comment:
Hi @olamy 

I was trying to track down where I have any of these options in my build as I started to see these warnigns in the logs 🙈 🙂 

It seems that with this change the message is logged all the time, no matter if any option is set or not ..

maybe it should've been:

```suggestion
        if (old && getLog().isWarnEnabled()) {
            getLog().warn("Javadoc 1.4+ doesn't support the -1.1 switch anymore. Ignore this option.");
}
```

Reply:
Sure see there https://github.com/apache/maven-javadoc-plugin/pull/1240

Reply:
nice 🙂 thanks for pointing me to that PR 🎉!